shared/storage/minio.py

```diff
@@ -95,7 +95,11 @@ def init_minio_client(
                 ),
             )
         return Minio(
-            host, access_key=access_key, secret_key=secret_key, secure=verify_ssl, region=region
+            host,
```

Comment:
the rest are lint

Reply:
ty, subtle but makes the review far easier